LICENSE

```diff
@@ -234,7 +237,7 @@
     1. Fast R-CNN  - For details, see example/rcnn/LICENSE
     2. Faster R-CNN - For details, see example/rcnn/LICENSE
     3. tree_lstm - For details, see example/gluon/tree_lstm/LICENSE
-
+    4. JQuery - For details, see http://jquery.org/license
```

Comment:
You should include the actual license text in the package, since it could vary between releases of jquery.  Use the same pointer file structure as other files mentioned in this section.

Reply:
ok